backend/app/Http/Controllers/Web/Admin/WorkOrderManagementController.php

```
@@ -235,11 +266,86 @@ public function show(WorkOrder $workOrder, CustomFieldService $customFields)
+                (float) $request->validated('qty'),
+                $request->user(),
+                $request->validated('reason'),
+            );
+
+            return back()->with('success', __('Material returned to stock'));
+        } catch (\DomainException|\InvalidArgumentException $e) {
+            return back()->with('error', $e->getMessage());
+        }
+    }
+
+    public function reclassify(ReclassifyClassRequest $request, WorkOrder $workOrder, MaterialReclassificationService $reclassifications)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$workOrder'. (undefined)

(UnusedFormalParameter)